app/server/api-generator/src/main/java/io/syndesis/server/api/generator/swagger/util/Oas20ModelParser.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.syndesis.server.api.generator.swagger.util;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.github.fge.jsonschema.core.exceptions.ProcessingException;
+import com.github.fge.jsonschema.core.load.configuration.LoadingConfiguration;
+import com.github.fge.jsonschema.core.report.ProcessingMessage;
+import com.github.fge.jsonschema.core.report.ProcessingReport;
+import com.github.fge.jsonschema.main.JsonSchema;
+import com.github.fge.jsonschema.main.JsonSchemaFactory;
+import io.apicurio.datamodels.Library;
+import io.apicurio.datamodels.core.models.Document;
+import io.apicurio.datamodels.openapi.v2.models.Oas20Document;
+import io.syndesis.common.model.Violation;
+import io.syndesis.common.util.IOStreams;
+import io.syndesis.common.util.Resources;
+import io.syndesis.common.util.json.JsonUtils;
+import io.syndesis.server.api.generator.APIValidationContext;
+import io.syndesis.server.api.generator.swagger.OpenApiModelInfo;
+import io.syndesis.server.api.generator.swagger.OpenApiValidationRules;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yaml.snakeyaml.Yaml;
+
+import static java.util.Optional.ofNullable;
+
+/**
+ * @author Christoph Deppisch
```

Comment:
I prefer code without `@author` tags.

Reply:
Yeah it's in my IDE settings from ages ago. Perhaps I need to rethink